shell_lint.sh

```diff
@@ -8,4 +8,5 @@ set -euo pipefail
 SHELLCHECK_VERSION=v0.8.0
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
-docker run --rm -t -v "$SCRIPT_DIR":"$SCRIPT_DIR" "koalaman/shellcheck:$SHELLCHECK_VERSION" -x $SCRIPT_DIR/scripts/*.sh
+# shellcheck disable=SC2046
+docker run --rm -t -v "$SCRIPT_DIR":"$SCRIPT_DIR" "koalaman/shellcheck:$SHELLCHECK_VERSION" -x $(git ls-tree -r HEAD --name-only | grep '.sh$' | sed "s,.*,$SCRIPT_DIR/&,")
```

Comment:
I use `git ls-tree` and not just `find | grep .sh` because I have many untracked `.sh` files that I don't want to remove :)